### PR TITLE
Article on Contributing Articles

### PR DESCRIPTION
**Pull request by Xilophor**
Created an article on how to contribute wiki pages. Goes over forking the repo, creating the article file, viewing the webpage in the dev environment, formatting, adding to the sidebar, and creating a PR.

- Content in article is based on organization updates in #51 
- Moved `translating-the-wiki` to `contribute` instead of `translation` so that new article can be under same directory.

### CODE REVIEW

**Comment by Xilophor** [comment]
Let me know if I'm missing any important information in the article

File: docs/contribute/writing-articles.md [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+
+![GitHub Fork Repo Button](/public/images/writing-articles/forkgithubrepobutton.png)
+
+![GitHub Create Fork Dialog](/public/images/writing-articles/forkgithubrepopopup.png)
+
+Now we've created the fork! From there you can make changes to a branch on the fork and create the PR when you're finished.
+
+## Creating the File
+
+Creating the article file is pretty simple. Add the file to the folder/subsection you are adding to, such as `/docs/dev/api/` for APIs, and create a markdown file (.md).
```
Comment (legoandmars) [inline docs/contribute/writing-articles.md:27]: i don't know how much it matters, but apis are technically in `/docs/apis`

File: docs/contribute/writing-articles.md [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+
+::: danger IMPORTANT
+Make sure you make your file name with no spaces or symbols, other than `-` to replace spaces!
+
+For example, this article is named `writing-articles.md` and is located in `/docs/extras/`.
+:::
+
+The terminal will then display the following, at which point you can go to the URL given in the terminal:
+
+![Displayed Text in Terminal](/public/images/writing-articles/runvitepressterminal.png)
```
Comment (legoandmars) [inline docs/contribute/writing-articles.md:37]: i think this accidentally got put in the wrong section, it should be in `Viewing Your Article`

File: docs/contribute/writing-articles.md [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+This must run in your repo folder. You can do so using the terminal in the code editor/IDE that you use. 
+:::
+
+## Frontmatter
+
+VitePress uses `YAML Frontmatter` for article configs. There are three commonly used settings, `description`, `prev`, and `next`.
+
+### Description
+
+In order to have descriptive info for search engines and embeds, you have to add a description to the front matter. To do so, you need the following code at the start of your article:
```
Comment (legoandmars) [inline docs/contribute/writing-articles.md:59]: maybe add a sentence that's like "This should ideally be short and descriptive (one to two sentences long)"

File: docs/contribute/writing-articles.md [hunk 4]
```diff
@@ -0,0 +1,400 @@
...
+
+- Using `#` will create the main heading (h1). For example, [this links to a main heading](#contributing-articles).
+- Using `##` will create the an h2 heading.
+
+This pattern continues until `######`, which creates an h6 heading.
+
+::: info
+Unless there is an empty line between lines of text, headings, or other line symbols, the lines of text, heading, code, or whatever else will either not work or be separated as intended.
+
+::: code-group
```
Comment (legoandmars) [inline docs/contribute/writing-articles.md:146]: this is not a change request this info box is just really cool

great work

File: docs/contribute/writing-articles.md [hunk 5]
```diff
@@ -0,0 +1,400 @@
...
+Images should be saved in a folder within `/docs/public/images`, where the folder is labeled the same as your article file name (minus `.md`).
+
+For example, all images used in this article are saved within `/docs/public/images/writing-articles`.
+:::
+
+::: info 2. Add it with the correct syntax
+The syntax to add an image is like adding a link.
+
+```md
+![Example Image Alt Text](/public/images/writing-articles/example-image.png)
```
Comment (legoandmars) [inline docs/contribute/writing-articles.md:325]: should be `/images/writing-articles/example-image.png`, public is unnecessary

**Review by legoandmars — APPROVED** [review]
image links need to be moved from `/public/images/...` to `/images/...` because the former makes vitepress angry
 
WindowsTerminal_towaa2pLaI

overall looks very good though. i'll approve because all the changes are relatively minor, feel free to merge whenever they're fixed

**Comment by legoandmars** [comment]
this also might not be strictly within the scope of this PR but the "Other Resources" tab has inconsistent casing.
we might want to do a pass over all article names to make them consistently uppercase 

firefox_63SPh8qz97

File: docs/contribute/writing-articles.md (hunk 1 above)
Comment (legoandmars) [reply docs/contribute/writing-articles.md:27]: nevermind, just checked the separation PR

File: docs/contribute/writing-articles.md (hunk 4 above)
Comment (Xilophor) [reply docs/contribute/writing-articles.md:146]: VitePress can do some cool stuff :)

**Review by legoandmars — APPROVED** [review]

**Review by legoandmars — APPROVED** [review]

**Pull request merged**